WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1150,6 +1166,20 @@ InviteStatusTentative       = background-color: @InviteStatusTentativeBgColor@;
 InviteStatusCounterBgColor  = #fff4d5
 InviteStatusCounter         = background-color: @InviteStatusCounterBgColor@; margin: @MailViewGutterSpace@ 0;
 InvCounter                  = padding:0 24px;
+ComposeViewContainer        = @PageBg@ @FullWidth@ @Conv2Bg@; box-sizing: border-box; padding: @MailViewGutterSpace@; padding-top: 0;
+ComposeViewTopHeader        = background-color: @AppC@;
+ComposeViewFromWrapper      = @Conv2Bg@
+ComposeViewToTd             = padding-top: @ComposeViewOutsidePadding@;
+ComposeViewSubjectInputField= border:none; color:@TxtC@; height:3rem; @FullWidth@ @BottomSeparator@ @FontSize-big@
+ComposeViewHeaderRowFirstTd = padding-left: @ComposeViewOutsidePadding@;
+ComposeViewHeaderButtonTitle= text-align: left; padding: 0; display:inline; line-height: 3em;
+ComposeViewTinyMceContainer = 
+ComposeViewHtmlEditor       = padding:@ComposeViewOutsidePadding@; background-color: @AppC@; 
+ComposeViewHeaderRows       =
+ComposeViewHeaderEmptyRight = padding-right: 18px;
```

Comment:
Cannot be used as we need specifically 18px only to align the inputs for to/cc/bcc with the tinymce or html editor in line